glide.yaml

```diff
@@ -28,7 +24,7 @@ import:
 - package: go.uber.org/thriftrw
   version: v1.14.0
 - package: github.com/uber/tchannel-go
-  version: 1.5.0
+  version: ^1.6.0
```

Comment:
comment here, thrift is coming tchannel-go

Reply:
Tally and Tchannel are the only dependencies which use apache/thrift and both pin apache/thrift at '>=0.9.3, <0.11.0'. Since zanzibar is not directly using apache/thrift, zanzibar has no reason to know about apache/thrift